src/plugins/intel_gna/src/optimizer/gna_pass_manager.cpp

```diff
@@ -62,6 +62,37 @@ std::shared_ptr<IPassManager> BasePass::getPassManager() {
     return sharedMgr;
 }
 
+static auto DoNotSkip = [](CNNLayerPtr) {
```

Comment:
1. What is the benefit of using lambda instead of function here?
2. I see that in pass manager we already have two exactly the same lambdas "donotSkip" lambdas.
Maybe it would be good to use one solution?

Reply:
Yes, I meant it to be a function. Changed also other occurrences.